src/tutorials/migrating.md

```diff
@@ -0,0 +1,54 @@
+---
+search:
+    keywords: ['migrating', 'upload']
+---
+
+# Migrating to Platform.sh
+
+Moving an already-built site to Platform.sh is generally straightforward.  For the most part, the only part that will vary from one framework to another is the details of the Platform.sh configuration files.  See the **Featured Framework** section or our [project templates](https://github.com/platformsh/) for more project-specific documentation.
```

Comment:
Ideally "Platform.sh configuration files" should be linked to something, but I notice we probably don't have a single page that can be linked (there are 3, for app, routes, and services).